protobuf-provider/src/main/java/io/confluent/kafka/schemaregistry/protobuf/ProtobufSchemaUtils.java

```diff
@@ -107,6 +110,80 @@ public static byte[] toJson(Message message) throws IOException {
     return jsonString.getBytes(StandardCharsets.UTF_8);
   }
 
+  public static ProtoFileElement jsonToFile(JsonNode node) throws JsonProcessingException {
+    ObjectMapper mapper = new ObjectMapper();
+    SimpleModule module = new SimpleModule();
+    module.addDeserializer(ProtoFileElement.class, new ProtoFileElementDeserializer());
+    mapper.registerModule(module);
+
+    String serialized = node.toString();
+    return mapper.readValue(serialized, ProtoFileElement.class);
```

Comment:
I believe you can pass the JsonNode directly to readValue, which is more efficient.  Let me know if this doesn't work as there are other ways to create an object from a JsonNode w/o going through the string repressentation